scripts/default-branch/issue-pr-templates-docs.php

```diff
@@ -36,9 +36,6 @@
 - [ ] CI is green
 EOT;
 
-// The template used when clicking "Open a blank issue"
-$defaultIssueTemplate = "<!-- Blank templates are for use by maintainers only! If you aren't a maintainer, please go back and pick one of the issue templates. -->";
```

Comment:
I don't think there a way to provide this information on blank issues any longer